package.json

```diff
@@ -14,13 +14,17 @@
       "url": "https://github.com/danielbeardsley/cimpler.git"
    },
    "dependencies": {
+      "body-parser": "^1.19.0",
       "connect": "~3.7.0",
       "github": "~3.1.1",
       "log4js": "~0.5.4",
       "notify-queue": "~0.0.5",
       "optimist": "~0.3.4",
+      "qs": "^6.7.0",
+      "response-time": "^2.3.2",
       "shell-quote": "^1.6.1",
-      "underscore": "~1.4.3"
+      "underscore": "~1.4.3",
+      "url": "^0.11.0"
```

Comment:
Is there a reason to not use the node.js built-in `url` module?

Reply:
I think I tried importing it and got an error about the file not being a module or something.

Reply:
Fixed! 88a923d